tests/unit_tests/test_dockerlauncher.py

```diff
@@ -90,15 +91,6 @@ def test_start(mocker, capsys, caplog, enshell_mock, tmpdir):
     values_run_command[0] = [0, "set_no_reroute_log"]
     enshell_mock[0].run_command.side_effect = values_run_command.copy()
     mocker.patch.object(enshell_grpc, "EnShellGRPC", return_value=enshell_mock[0])
-    dock = mocker.patch.object(docker, "from_env", return_value=docker_client)
```

Comment:
In the PR I removed the check for the docker module.
We are always installing it, there's no need to check if the module loaded or not. 
The docker module is then used only if DockerLauncher is started, or docker is asked for in LibUserd. If docker is not installed on the machine where these are being asked for, it is correct that the docker-py module should fail miserably.
So these checks are no more required